test/services/katello/managed_content_medium_provider.rb

```diff
@@ -0,0 +1,25 @@
+require 'katello_test_helper'
+
+module Katello
+  module Service
+    class ManagedContentMediumProviderTestBase < ActiveSupport::TestCase
+    end
+
+    class ManagedContentMediumProvider < ManagedContentMediumProviderTestBase
+      setup do
+        @org = FactoryBot.create(:katello_organization)
+        @distro = katello_repositories(:fedora_17_x86_64)
```

Comment:
We can use [`FactoryBot`](https://github.com/Katello/katello/blob/master/test/factories/repository_factory.rb#L11) here too.

Reply:
Let's try. I thought, we depended on the inner plumbings of that Respository.